src/zeroband/inference/toploc.py

```diff
@@ -129,3 +140,49 @@ def wait_for_proofs(self):
             return
 
         wait(list(i for i in self._proof_futures.values() if i is not None))
+
+
+def toploc_hook(_, inputs: tuple, toploc: TOPLOC):
```

Comment:
```suggestion
def toploc_hook(_, inputs: tuple, toploc: TOPLOC):
```
_ this might break passing by kwargs at some point ?

Wondering if we should rather do *args,**kwargs, this would be more pythonic

Reply:
basically the first two arguments to a pre-hook are `module` and `inputs`. the first one we don't need right now, hence the underscore. the toploc arg is just there to access it as a local var but we need to partially apply the function to pass as a hook. i'm not sure we will run into the issue you point out. the only danger is if inputs changes with a vllm bump, but there is nothing really we can do except for paying close attention to how their interface changes (and have tests that catch this)